noxfile.py

```diff
@@ -29,13 +29,13 @@ def test(s: Session) -> None:
 @session(venv_backend="none")
 def fmt(s: Session) -> None:
     s.run("ruff", "check", ".", "--select", "I", "--fix")
-    s.run("black", ".")
+    s.run("ruff", "format", ".")
 
 
 @session(venv_backend="none")
 def fmt_check(s: Session) -> None:
     s.run("ruff", "check", ".", "--select", "I")
-    s.run("black", "--check", ".")
+    s.run("ruff", "format", "--check", ".")
```

Comment:
With this design, if the the first check fails, no errors for the second check will be reported. You can use [nox parameterization](https://github.com/drhagen/tensora/blob/master/noxfile.py#L34-L37) to always run both checks. It is admittedly, a little more complicated, but this is the way to do it unless Ruff merges `format --check` into `check`.